cmdutil/https/serve.go

```diff
@@ -20,8 +21,10 @@ func Serve(handler http.Handler, cfg Config, opts ...func(*http.Server)) error {
 		serverCert = []byte(cfg.ServerCert)
 		serverKey  = []byte(cfg.ServerKey)
 		srv        = &http.Server{
-			Addr:    fmt.Sprintf(":%d", cfg.SecurePort),
-			Handler: handler,
+			Addr:         fmt.Sprintf(":%d", cfg.SecurePort),
+			Handler:      handler,
+			WriteTimeout: cfg.WriteTimeout * time.Second,
```

Comment:
If WriteTimeout and ReadTimeout are already time.Duration the multiply by time.Second can be skipped.

Reply:
Ah yes good catch, I'll address that.